README.md

```diff
@@ -1 +1,5 @@
-# Review-one-of-your-past-projects
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Documentation is an important part of project development. It could be better if you add a README file that stick to the [README template](https://github.com/microverseinc/readme-template) given by microverse, and you should customize it according to the project